release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -48,12 +48,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-10-installation-and-upgrade"]
 === Installation and upgrade
 
-[id="ocp-4-10-installation-and-upgrade-aws-ami"]
 ==== Installing a cluster into an Amazon Web Services GovCloud region
 
 {op-system-first} Amazon Machine Images (AMIs) are now available for AWS GovCloud regions. The availability of these AMIs improves the installation process because you are no longer required to upload a custom RHCOS AMI to deploy a cluster.
 //For more information on installing to a AWS GovCloud region, <insert link to topic after it merges>
 
+[id="ocp-4-10-installation-and-upgrade-tagrole"]
+==== Using a custom AWS IAM role to deploy an installer-provisioned cluster
+
+Beginning with {product-title} {product-version}, the installer does not require the AWS `iam:TagRole` permission to create the IAM role when deploying a cluster. This enhancement improves the installation process for organizations that want to use a custom IAM role, but whose security policies prevent the use of `iam:TagRole`.
+
+Previously, if the IAM user deploying the cluster was assigned to a custom role, the installer added the `shared` tag to it. Adding this tag required the `iam:TagRole` permission.
```

Comment:
@mjpytlak  same as https://github.com/openshift/openshift-docs/pull/40398#issuecomment-1014086288, IMO, we may only mention **_installer will not add tag to byo IAM role_**, and do not mention the permission `iam:TagRole`, WDYT?

Reply:
Yes @yunjiang29 now that I better understand the nuances, I agree. Thoughts on my update?